.docker/entrypoint/spec/jekyll_commander_spec.rb

```diff
@@ -81,27 +87,24 @@
 
       before(:all) do
         @thread = Thread.new do
-          jekyll_config_provider = Jekyll::PlantUml::JekyllConfigProvider.new(data_dir)
+          exec_env = ExecEnv.new('development', __dir__, data_dir)
+          jekyll_config_provider = JekyllConfigProvider.new(exec_env, :info)
           jekyll_config = jekyll_config_provider.provide('serve')
-          jekyll_commander = Jekyll::PlantUml::Commands::JekyllCommander.new(jekyll_config)
+          jekyll_commander = JekyllCommander.new(jekyll_config, :info)
           jekyll_commander.execute('serve')
         end
         @thread.abort_on_exception = true
 
-        Jekyll::Commands::Serve.mutex.synchronize do
-          unless Jekyll::Commands::Serve.running?
-            Jekyll::Commands::Serve.run_cond.wait(Jekyll::Commands::Serve.mutex)
-          end
+        JekyllServe.mutex.synchronize do
+          JekyllServe.run_cond.wait(JekyllServe.mutex) unless JekyllServe.running?
```

Comment:
Codacy found an issue: [Modifier form of `unless` makes the line too long.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)